app/src/organisms/Devices/ProtocolRun/SetupModules/SetupModulesList.tsx

```diff
@@ -252,76 +254,100 @@ export const ModulesListItem = ({
       </StyledText>
     )
   }
+
+  const RenderModuleStatus = (): JSX.Element => {
+    const handleCalibrate = (): void => {
+      setShowModuleWizard(true)
+    }
+    if (attachedModuleMatch == null) {
+      return (
+        <StatusLabel
+          id={location}
+          status={moduleConnectionStatus}
+          backgroundColor={COLORS.warningBackgroundLight}
+          iconColor={COLORS.warningEnabled}
+          textColor={COLORS.warningText}
+        />
+      )
+    } else if (attachedModuleMatch.moduleOffset?.last_modified != null) {
+      return (
+        <StatusLabel
+          id={location}
+          status={moduleConnectionStatus}
+          backgroundColor={COLORS.successBackgroundLight}
+          iconColor={COLORS.successEnabled}
+          textColor={COLORS.successText}
+        />
+      )
+    } else {
+      return (
+        <TertiaryButton onClick={handleCalibrate}>
+          {t('calibrate_now')}
+        </TertiaryButton>
+      )
+    }
+  }
+
   return (
-    <Box
-      border={BORDERS.styleSolid}
-      borderColor={COLORS.medGreyEnabled}
-      borderWidth="1px"
-      borderRadius={BORDERS.radiusSoftCorners}
-      padding={SPACING.spacing16}
-      backgroundColor={COLORS.white}
-      data-testid="ModulesListItem_Row"
-    >
-      {showHeaterShakerFlow && heaterShakerModuleFromProtocol != null ? (
-        <HeaterShakerWizard
-          onCloseClick={() => setShowHeaterShakerFlow(false)}
-          moduleFromProtocol={heaterShakerModuleFromProtocol}
-          attachedModule={heaterShakerAttachedModule}
+    <>
+      {showModuleWizard && attachedModuleMatch != null ? (
```

Comment:
```suggestion
      {showModuleWizard  ? (
```
Is the attachedModuleMatch check necessary here? The only way we can show the wizard is if the button is clicked?

Reply:
I added `attachedModuleMatch != null` because of `ModuleWiardFlowProps`.

```
interface ModuleWizardFlowsProps {
  attachedModule: AttachedModule
  closeFlow: () => void
  slotName?: string
  onComplete?: () => void
}
```

Actually I wrote `{showModuleWizard ? (` first and got an error.